components/TradingCard.jsx

```diff
@@ -0,0 +1,26 @@
+import Link from 'next/link';
+import HoursBar from './HoursBar';
+import TradingCardImage from './TradingCardImage';
+
+export default function TradingCard({
+  photo, forecastedHours, name, jobTitle, id, weeklyCapacity,
+}) {
+  return (
+    <div className="cmp-trading-card">
+      <TradingCardImage
+        imageUrl={photo}
+        weeklyCapacity={weeklyCapacity}
+        forecastedHours={forecastedHours}
+      />
+      <div className="cmp-trading-card__hrs-bar">
+        <HoursBar forecastedHours={forecastedHours} weeklyCapacity={weeklyCapacity} />
+      </div>
+      <Link href={`/${id}`} passHref>
+        <a href="replace">
+          <p className="cmp-trading-card__name">{name}</p>
+        </a>
```

Comment:
One UX recommendation: we could use an `::after` pseudo element to expand the clickable area of the link over the image. This would involve some `position`-ing, but would help make the cards easier to use.

Reply:
This sort of explains the idea (among a lot of other concerns about cards): https://inclusive-components.design/cards/#thepseudocontenttrick

Reply:
My most recent push has a solution that does this now.